andela/bp-esa-backend
# #162192779 Throws an error if environment variables are not configured

#### What does this PR do?
Throws an error if the environment variables are not configured

#### Description of Task to be completed?
Add check on the modules that throws an error if one of the required variables is not there

#### How should this be manually tested?
- Checkout to this branch.
- Remove the slack token from the env file
- Run the app using `npm start`.
- An error should be thrown saying `Some variables are missing`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#162192779](https://www.pivotaltracker.com/story/show/162192779)

#### Screenshots (If applicable)
N/A
